<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemDialog</class>
 <widget class="QDialog" name="ItemDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>410</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>410</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>410</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>New item</string>
  </property>
  <property name="windowIcon">
   <iconset resource="FortResources.qrc">
    <normaloff>:/icons/Fort255.png</normaloff>:/icons/Fort255.png</iconset>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>370</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>381</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>Optional</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="QLineEdit" name="lineEditUrl">
    <property name="geometry">
     <rect>
      <x>79</x>
      <y>30</y>
      <width>291</width>
      <height>27</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>Optional url for the website these credentials are used</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelUrl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>24</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Url:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxTagAsFavorite">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Tag as favorite</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Essential</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>361</width>
      <height>129</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="labelTitle_2">
       <property name="text">
        <string>Title:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineTitle_2"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelUser">
       <property name="text">
        <string>User:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineUser"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelPassword">
       <property name="text">
        <string>Password:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="linePassword">
       <property name="inputMask">
        <string/>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>Show password</string>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>240</y>
     <width>291</width>
     <height>91</height>
    </rect>
   </property>
   <property name="lineWrapMode">
    <enum>QTextEdit::NoWrap</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Notes:</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineTitle_2</tabstop>
  <tabstop>lineUser</tabstop>
  <tabstop>linePassword</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>lineEditUrl</tabstop>
  <tabstop>textEdit</tabstop>
 </tabstops>
 <resources>
  <include location="FortResources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ItemDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ItemDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
